Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
-# create a work dir and copy local files in
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/console-operator
 COPY . .
-RUN make build
+RUN ADDITIONAL_GOTAGS="ocp" make build WHAT="cmd/console"
 
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+FROM registry.svc.ci.openshift.org/ocp/4.0:base
```

Comment:
Not sure if we should change the base image. Let's check on that.

Reply:
Also be mindful of #187 which is going to merge soon with some tweaks to the Dockerfile.